input/groups/weapon.json

```diff
@@ -243,7 +285,28 @@
         4027,
         3963
       ],
-      "Maguuma Jungle Region: Glyphic": [
+      "Cultural: Norn Tier 3 - Wolfborn": [
+        4293,
+        4294,
+        4297,
+        4298,
+        4334,
+        4302,
+        4303,
+        4304,
+        4272,
+        4305,
+        4275,
+        4276,
+        4342,
+        4344,
+        4345,
+        4315,
+        4283,
+        4316,
+        4286
+      ],
+      "Region: Tarnished Coast / Cultural: Asura Tier 1 / Cultural: Sylvari Tier 1 - Glyphic": [
```

Comment:
This kind of underlines the point, there's no way these skins are culturally significant to both asura and sylvari. It's just that the cultural vendor is selling something from their shared region.

Reply:
They're technically both, the vendors are clearly defined as "Cultural Weaponsmith [tier 1/2/3]", just like the armor vendors. See here: https://wiki.guildwars2.com/wiki/Cultural_weaponsmith

If you want to prioritize one over another, I'd also put region first, but then remove any mention of tiers from the cultural weapon sets (so, just "Cultural: Asura" for the t3, with no mention of t3). Why? Because we have the Ebonhawke and Lion's Arch precedents, where they also have cultural weaponsmiths, but they're the same skin all across three tiers.

There's only one condition to this change, and it's integrating those other three non-racial cultural weaponsmiths as well. They don't have a parent race, so they would have to follow an "other" format, such as "Cultural: Other - Lionguard".

I think this approach would solve most of your issues, all while being true to the game's own classification :D.